src/test/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJobTest.java

```diff
@@ -83,7 +87,19 @@
     public void allAnnotationStepsShouldBeExecuted() throws Exception {
         mongoRule.restoreDump(getResourceUrl(MONGO_DUMP), jobOptions.getDbName());
 
-        JobExecution jobExecution = jobLauncherTestUtils.launchJob();
+        JobParameters jobParameters = new EvaJobParameterBuilder()
+                .vepPath(getResource(MOCK_VEP).getPath())
+                .vepCacheVersion("")
+                .vepCachePath("")
+                .vepCacheSpecies("")
+                .inputFasta("")
+                .vepNumForks("")
+                .outputDirAnnotation(OUTPUT_DIR_ANNOTATION)
+                .inputStudyId(INPUT_STUDY_ID)
+                .inputVcfId(INPUT_VCF_ID)
```

Comment:
It would be helpful for reading if these calls could be sorted alphabetically. Applies to the other tests classes.